### PR TITLE
Add optional additional_disk input to SortSam task

### PR DESCRIPTION
**Pull request by timaeusx**

### CODE REVIEW

**Review by khajoue2 — APPROVED** [review]

File: pipelines/broad/dna_seq/germline/single_sample/wgs/WholeGenomeGermlineSingleSample.changelog.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+# 3.1.6
+
```
Comment (khajoue2) [inline pipelines/broad/dna_seq/germline/single_sample/wgs/WholeGenomeGermlineSingleSample.changelog.md:2]: remove this space after the version.

File: pipelines/broad/dna_seq/germline/variant_calling/VariantCalling.changelog.md [hunk 2]
```diff
@@ -1,3 +1,9 @@
+# 2.1.5
+
```
Comment (khajoue2) [inline pipelines/broad/dna_seq/germline/variant_calling/VariantCalling.changelog.md:2]: remove this empty line.

File: pipelines/broad/reprocessing/exome/ExomeReprocessing.changelog.md [hunk 3]
```diff
@@ -1,3 +1,9 @@
+# 3.1.5
+
```
Comment (khajoue2) [inline pipelines/broad/reprocessing/exome/ExomeReprocessing.changelog.md:2]: remove empty line

File: pipelines/broad/reprocessing/external/exome/ExternalExomeReprocessing.changelog.md [hunk 4]
```diff
@@ -1,3 +1,9 @@
+# 3.1.7
+
```
Comment (khajoue2) [inline pipelines/broad/reprocessing/external/exome/ExternalExomeReprocessing.changelog.md:2]: same: remove empty line

**Review by khajoue2 — APPROVED** [review]

**Review by nikellepetrillo — APPROVED** [review]

**Pull request merged**